cdap-security/src/main/java/co/cask/cdap/security/authorization/DefaultAuthorizationEnforcementService.java

```diff
@@ -77,14 +67,6 @@ public void enforce(EntityId entity, Principal principal, Set<Action> actions) t
     if (!isSecurityAuthorizationEnabled()) {
       return;
     }
-    // For accessing system datasets for internal operations like recording metadata, usage, lineage, etc
-    if (Principal.SYSTEM.equals(principal)) {
-      return;
-    }
-    // If the principal is a superuser, allow access
-    if (isSuperUser(principal)) {
-      return;
-    }
 
     doEnforce(entity, principal, actions, true);
```

Comment:
not your PR but enforce itself is a verb we don't need to add do to make it a verb for method name.